packages/@aws-cdk/aws-sns/lib/subscription.ts

```diff
@@ -85,6 +92,7 @@ export class Subscription extends Resource {
       topicArn: props.topic.topicArn,
       rawMessageDelivery: props.rawMessageDelivery,
       filterPolicy: this.filterPolicy,
+      region: props.region,
```

Comment:
Just to make sure, if `region` is the same as the current region, this will still work, correct? Otherwise we might need some exception here.

Reply:
Correct. I was able to validate this by deploying a template.